charts/druid/templates/historical/statefulset.yaml

```
@@ -155,7 +155,7 @@ spec:
       spec:
         accessModes:
           - ReadWriteOnce
-        storageClassName: {{ default "standard" $historicalDefaults.persistence.storageClass }}
+        storageClassName: {{ default "" $historicalDefaults.persistence.storageClass }}
```

Thanks!

To be completely clean, I would suggest:
```suggestion
        {{ default nil $historicalDefaults.persistence.storageClass }}
```

So that reading it does not cause confusion with an "empty string storageClassName", which is different.

Moreover, could you bump version in Chart.yaml?